lib/matplotlib/widgets.py

```diff
@@ -2418,25 +2435,43 @@ def _release(self, event):
 
     def _onmove(self, event):
         """Motion notify event handler."""
-        # resize an existing shape
-        if self.active_handle and self.active_handle != 'C':
-            x0, x1, y0, y1 = self._extents_on_press
-            if self.active_handle in ['E', 'W'] + self._corner_order:
-                x1 = event.xdata
-            if self.active_handle in ['N', 'S'] + self._corner_order:
-                y1 = event.ydata
+        # Current rectangle properties
+        x0, y0, width, height, rotation = self._rect_properties
+        # Current mouse x, y coord
+        xmove = event.xdata
+        ymove = event.ydata
+
+        # rotate existing shape
+        if 'rotate' in self.state:
+            rotation_new = np.arctan2(ymove - y0,
+                                      xmove - x0)
+            rotation_old = np.arctan2(self.prev_ymove - y0,
+                                      self.prev_xmove - x0)
+            rotation += rotation_new - rotation_old
 
         # move existing shape
-        elif (('move' in self.state or self.active_handle == 'C' or
-               (self.drag_from_anywhere and self._contains(event))) and
```

Comment:
What happened to `drag_from_anywhere` support?

Reply:
Woops, I think this got lost in the rebase I had to do.

Reply:
Woops, got lost in a rebase, should be back now.